spec/jobs/send_membership_request_mailer_job_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "./app/jobs/memberships/send_membership_request_mailer_job.rb"
+
+describe SendMembershipRequestMailerJob do
+  let(:type) { { email_type: email_type } }
+  let(:recipient_info) { { email: email } }
+  let(:email_type) { " " }
+  let(:email) { "bob.schmidt@va.gov" }
+  let(:name) { "Bob" }
+
+  let(:perform_job) do
+    SendMembershipRequestMailerJob.new(type: type, recipient_info: recipient_info)
```

Comment:
```suggestion
    Memberhships::SendMembershipRequestMailerJob.new(type, recipient_info)
```

In order to mesh with a suggestion made in `app/jobs/memberships/send_membership_request_mailer_job.rb`, the keyword arguments will need to be changed to positional ones.